lib/banktrans.rb

```diff
@@ -0,0 +1,21 @@
+# Challenge 3 - Bank Transactions
+
+operation = gets.chomp
```

Comment:
This challenge is not done, i would suggest you could look it up again over the weekend